users/ibnuda/config.h

```diff
@@ -0,0 +1,10 @@
+#pragma once
+
+#define COMBO_COUNT 18
+#define COMBO_TERM 100
+
+#define IGNORE_MOD_TAP_INTERRUPT
+#define PERMISSIVE_HOLD
+
+#define TAPPING_TERM 200
+#define TAPPING_TERM 200
```

Comment:
```suggestion
```

`#define TAPPING_TERM 200` is here twice.

Reply:
you're right. it was past midnight when i commit the file.